### PR TITLE
WRP-13735: Used fake time as a CLI option

### PR DESCRIPTION
**Pull request by ion-andrusciac-lgp**
### Checklist 

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Use fake time as a CLI option

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Created a server in the `console`  app that can enable/disable fake-time

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
After `npm run serve` it will be needed to run `npm run fakeTime` and from the terminal, fake-time option can be enabled/disabled
image

### Links
[//]: # (Related issues, references)
WRP-13735

### Comments

Enact-DCO-1.0-Signed-off-by: Ion Andrusciac ion.andrusciac@lgepartner.com

### CODE REVIEW

File: console/src/App/App.js [hunk 1]
```diff
@@ -220,6 +221,7 @@ const AppBase = kind({
 		showWelcomePopup,
 		skinName,
 		userId,
+		fakeTimeEnabled,
```
Comment (daniel-stoian-lgp) [inline console/src/App/App.js]: please sort

**Review by daniel-stoian-lgp — COMMENTED** [review]
only one comment

**Review by daniel-stoian-lgp — APPROVED** [review]

File: console/src/App/fakeTimeServer.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+http.listen(3002, () => {
+	runTerminal();
+});
+
+io.on('connection', socket => {
+	setInterval(() => {
+		socket.emit('FAKE_TIME', enabled);
+	}, 1000)
+});
```
Comment (stanca-pop-lgp) [inline console/src/App/fakeTimeServer.js]: Please add new line at the end of this file.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM. Great job!

**Pull request merged**